provision/acc_provision/templates/provision-config.yaml

```diff
@@ -2,21 +2,29 @@
 # Configuration for ACI Fabric
 #
 aci_config:
-  system_id: mykube             # Every opflex cluster must have a distict ID
+  system_id: mykube              # Every opflex cluster must have a distict ID
   apic_hosts:                   # List of APIC hosts to connect for APIC API
   - 10.1.1.101
-  vmm_domain:                   # Kubernetes container domain configuration
+  vmm_domain:                   # Kubernetes VMM domain configuration
     encap_type: vxlan           # Encap mode: vxlan or vlan
+#    encap_type: vlan           # Encap mode: vxlan or vlan
+#    vlan_range:
+#      start: 200
+#      end: 209
     mcast_range:                # Every opflex VMM must use a distinct range
-      start: 225.20.1.1
-      end: 225.20.255.255
-    nested_inside:              # Include if nested inside a VMM; required for CloudFoundry
-    # type: vmware              # Specify the VMM vendor (supported: vmware)
-    # name: myvmware            # Specify the name of the VMM domain
+      start: 225.21.1.1
+      end: 225.21.255.255
+    nested_inside:
+    #  type: vmware
+    #  name: myvmware
+  cloudfoundry:                 # CloudFoundry specific config parameters
+    custom_epg: ["cf-group1", "cf-group2"]
+    isolated_segment: True
```

Comment:
Need to support multiple IS. I think a list of dicts like this is needed:
cloudfoundry:
[..]
    isolated_segments:
        - name: foo                           # required
           subnet: 1.2.3.4/28             # required
        - name: ...
           subnet: ...
Also, these values need to be validated in config_validate()